sonar/modules/documents/dojson/rerodoc/model.py

```diff
@@ -762,7 +764,7 @@ def marc21_to_classification_field_084(self, key, value):
 @utils.for_each_value
 @utils.ignore_value
 def marc21_to_content_note(self, key, value):
```

Comment:
You can probably use: https://github.com/inveniosoftware/dojson/blob/5093f4d6bb15654d98af3edfa5367b4a84e5a45b/dojson/utils.py#L46